index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <title>David Leiva</title>
+  </head>
+  <body>
+
+    <!-- Main navigation bar or toolbar -->
+    <nav>
+      <div class="logo"><a href="#">My Logo</a></div>
+      <div class="menu-nav">
+        <button class="hamburger" type="button"><img src="img/icon-hamburger.png" alt="Menu icon"></button>
+        <ul class="links">
+          <li><a>Portfolio</a></li>
+          <li><a>About</a></li>
+          <li><a>Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <!-- Headline Section -->
+    <header>
+      <h1>I’m David.<br>Glad to see<br>you!</h1>
+      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <a class="connect">LET'S CONNECT</a>
+      <ul>
+        <li><a href="https://twitter.com/DavidLeiva111"><img src="img/twitter-icon.png" alt="Twitter icon" class="icon"/></a></li>
+        <li><a href="https://linkedin.com/in/1259221-d-leiva"><img src="img/linkedin-icon.png" alt="LinkedIn icon" class="icon"/></a></li>
+        <li><a><img src="img/medium-icon.png" alt="Medium icon" class="icon"/></a></li>
+        <li><a href="https://github.com/MasterLeiva"><img src="img/github-icon.png" alt="GitHub icon" class="icon"/></a></li>
+        <li><a><img src="img/angelist-icon.png" alt="Angelist icon" class="icon"/></a></li>
+      </ul>
+    </header>
```

Comment:
- [ ] I appreciate your efforts in adding the heading section to your project. However, instead of using the `<header>` tag, it would be more appropriate to use the `<section>` tag according to the HTML structure guidelines. 
- [ ] The recommended HTML structure is as follows:

![Screenshot 2023-06-02 033219](https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/118761303/9261148b-c7de-41f1-8ea4-6865394ba235)

- [ ]  By following this structure, you can ensure a well-organized and semantically correct HTML document. You can read [this](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure) article for more insights.